llvm/test/Transforms/JumpTableToSwitch/lit.local.cfg

```diff
@@ -0,0 +1,5 @@
+if bool(config.enable_profcheck):
+    # disable indirect call annotations here. The targets for JumpTableToSwitch
+    # are specific, and the profile injector is (intentionally) naive in what
+    # VP metadata it inserts.
+    config.substitutions.append(("opt", "opt -profcheck-annotate-indirect-calls=0"))
```

Comment:
nit: `=false` since it's a bool. Just reading the line here is confusing for the reader.